README.md

```diff
@@ -146,6 +146,34 @@ WMCO is not responsible for Windows operating system updates. The cluster admini
 creating the VMs and hence, the cluster administrator is responsible for providing an updated image. The cluster 
 administrator can provide an updated image by changing the image in the MachineSet spec.
 
+## Applying autoscaling to WMCO-managed Windows Machines
+
+There are two components to managing resources using autoscaling: a ClusterAutoscaler and MachineAutoscalers.
+The cluster autoscaler is capable of automatically adjust the size of the cluster based on node resource usage
+and overall deployment needs, within specific cluster constraints. Machine autoscalers modify machine sets
+to scale up and down the number of machines.
+Multiple MachineAutoscalers can be deployed to manage multiple MachineSets.
+Each machine autoscaler object should be mapped to a single machine set.
+
+- Define and deploy a [ClusterAutoscaler](https://docs.openshift.com/container-platform/4.6/machine_management/applying-autoscaling.html#configuring-clusterautoscaler).
```

Comment:
for my own education; does it only applies to version `4.6`  ? 

Otherwise, `latest` may help. See:  https://docs.openshift.com/container-platform/latest/machine_management/applying-autoscaling.html#configuring-clusterautoscaler

Reply:
Nope, you are correct -- also applies to 4.7/4.8. I'll update to latest.